migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,81 @@
+[id="migration-from-version-3-to-4-overview"]
+= Migration from version 3 to 4 overview
+include::modules/common-attributes.adoc[]
+:context: migration-from-version-3-to-4-overview
+
+toc::[]
+
+{product-title} 4 clusters are different from {product-title} 3 clusters. {product-title} 4 clusters contain new technologies and functionality that result in a cluster that is self-managing, flexible, and automated. To learn more about migrating from {product-title} 3 to 4 see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from OpenShift Container Platform 3 to 4].
+
+[id="mtc-3-to-4-overview-differences-mtc"]
+=== Differences between {product-title} 3 and 4
+Before migrating from {product-title} 3 to 4, you can check xref:../migrating_from_ocp_3_to_4/planning-migration-3-4.adoc#planning-migration-3-4[differences between {product-title} 3 and 4]. Review the following information:
+
+** Architecture
+** Installation and upgrade
+** Storage, network, logging, security, and monitoring considerations
+
+[id="mtc-3-to-4-overview-planning-network-considerations-mtc"]
+=== Planning network considerations
+Before migrating from {product-title} 3 to 4, review the following information on networking considerations:
+
+** xref:../migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc#dns-considerations_planning-considerations-3-4[DNS considerations]
+*** xref:../migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc#migration-isolating-dns-domain-of-target-cluster-from-clients_planning-considerations-3-4[Isolating the DNS domain of the target cluster from the clients].
+*** xref:../migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc#migration-setting-up-target-cluster-to-accept-source-dns-domain_planning-considerations-3-4[Setting up the target cluster to accept the source DNS domain].
+
+You can migrate stateful application workloads from {product-title} 3 to 4 at the granularity of a namespace. To learn more about MTC see xref:../migrating_from_ocp_3_to_4/about-mtc-3-4.adoc#about-mtc-3-4[Understanding MTC].
+
+[NOTE]
+====
+If you are migrating from {product-title} 3, see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from {product-title} 3 to 4] and xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Installing the legacy {mtc-full} Operator on {product-title} 3].
+====
+
+[id="mtc-overview-install-mtc"]
+== Installing MTC
+Review the following tasks to install the MTC:
+
+. Install the {mtc-full} Operator on target cluster by using Operator Lifecycle Manager (OLM).
+. Install the legacy {mtc-full} Operator on source cluster manually.
```

Comment:
Without actually reading any of the previous links, I'm a bit surprised that the user needs two clusters. Should there be another step before line 37?

Reply:
@XinRedhat Can you please confirm if we need one more step here?

Reply:
that's all for MTC installation.

Reply:
Thanks for confirming

Reply:
Addressed